.vscode/launch.json

```diff
@@ -4,6 +4,16 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387

```

Comment:
Does the comment type work in JSON?

Reply:
vscode seems to handle the comment without issue, do we think it should be removed?

Reply:
Ah ok, in that case I think it is fine